ddebug/pserror.h

```diff
@@ -161,87 +163,81 @@ void AssertionFailed(const char *expstr, const char *file, int line);
 // Brings up an error message for an int3
 void Int3MessageBox(const char *file, int line);
 //	Message box functions
+
 #define MBOX_OK 1
 #define MBOX_YESNO 2
 #define MBOX_YESNOCANCEL 3
 #define MBOX_ABORTRETRYIGNORE 4
+
 //	prints out a standard OS messagebox
 void OutrageMessageBox(const char *str, ...);
 int OutrageMessageBox(int type, const char *str, ...);
 // Sets the title for future OutrageMessageBox() dialogs
 void SetMessageBoxTitle(const char *title);
 // Write a block of text to the system clipboard
 void DumpTextToClipboard(char *text);
+
 //////////////////////////////////////////////////////////////////////////////
 //	development debugging functions
 //	adds a function to be called when a debug break occurs.
 //	undefine any ASSERT macro previously defined.
 #ifdef ASSERT
 #undef ASSERT
 #endif
+
 //	this callback is invoked when a DEBUG_BREAK macro is used.
 //		arguments
 //			style = 1 if ASSERT
 //					= 0 if Int3 debugger break.
 extern void (*DebugBreak_callback_stop)();
 extern void (*DebugBreak_callback_resume)();
+
 //	set DEBUG_BREAK callback
 static inline void SetDebugBreakHandlers(void (*stop)(), void (*resume)()) {
   DebugBreak_callback_stop = stop;
   DebugBreak_callback_resume = resume;
 }
-// DEBUG_BREAK()
-// Calls the debug_break() macro surrounded by calls to the debug callbacks (to turn off & on graphics)
-// ASSERT()
-// Like the standard C assert(), but if the condition failed and debugging on,
-// does a DEBUG_BREAK().  If debugging on, brings up a dialog.
-// Int3()
-// Does a DEBUG_BREAK() if debugging is turned on.  Also does an mprintf().
-//	Define the macros
+
 #ifndef RELEASE
 
 #include <cstdlib>
 
+// Calls the SDL_assert(false) macro surrounded by calls to the debug callbacks (to turn off & on graphics)
 #define DEBUG_BREAK()                                                                                                  \
   do {                                                                                                                 \
     if (DebugBreak_callback_stop)                                                                                      \
       (*DebugBreak_callback_stop)();                                                                                   \
-    debug_break();                                                                                                     \
+    SDL_assert(false);                                                                                                 \
     if (DebugBreak_callback_resume)                                                                                    \
       (*DebugBreak_callback_resume)();                                                                                 \
   } while (0)
 
+// Does a DEBUG_BREAK() if debugging is turned on.
 #define Int3()                                                                                                         \
```

Comment:
while we're renaming stuff, we should also rename `Int3` to something like `abort` (you can suggest a better name), Int3 is an assembly instruction without much semantic

Reply:
There too much `Int3()` usage, PR would be much wider. If this OK, I'd propose `ABORT()`

Reply:
I'm fine with `ABORT`